nerfstudio/data/utils/nerfstudio_collate.py

```diff
@@ -23,7 +23,12 @@
 
 import torch
 import torch.utils.data
-from torch._six import string_classes
+
+try:
+    from torch._six import string_classes
+except ImportError:
+    # _six deprecated in torch v2
+    string_classes = (str, bytes)
```

Comment:
Can we just edit ln 127 to `elif isinstance(elem, (str, bytes)):`?

Reply:
Sure